foodanddrink/restaurant/cron.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+
+from django.core.mail import EmailMessage
+
+import environ
+from django.template.loader import render_to_string
+
+from foodanddrink.foodanddrink.settings import env
+from foodanddrink.restaurant.models import Order, Customer
+
+env = environ.Env()
+environ.Env.read_env()
+
+
+def monthly_report():
+    mail_subject = "Monthly Report"
+    income = 0
+    now = datetime.now()
+
+    year = now.year
+    last_month = now.month - 1
+    if last_month == 0:
+        last_month = 12
+        year -= 1
+
+    orders = Order.objects.filter(date_ordered__year = year, date_ordered__month = last_month, status='f')
+    for order in orders:
+        income += order.total_price
```

Comment:
sử dụng query tính sum luôn thay vì xử lý vòng for sau khi query như hiện tại

Reply:
ok chị